splinterd/man/splinterd.1.md

```diff
@@ -1,81 +1,278 @@
-% COMMAND(1) Cargill, Incorporated | Splinter Commands
-
-<!--
-Template for a man page source file in Markdown format. The file should have the
-name of the command plus any subcommands (with dashes) with an extension of
-`1.md`. For example `splinter-cert-generate.1.md` is the Markdown file that
-generates the man page for `splinter cert generate`.
--->
+% SPLINTERD(1) Cargill, Incorporated | Splinter Commands
+
 NAME
 ====
 
-**command-subcommand** — About blurb from command help
+**splinterd** — Starts the Splinter daemon
 
 SYNOPSIS
 ========
-| **command** \[**FLAGS**\] \[**OPTIONS**\]
+
+**splinterd** \[**FLAGS**\] \[**OPTIONS**\] <arguments>
 
 DESCRIPTION
 ===========
-Paragraph or two that describes what the command does, explains how it fits in
-the Splinter environment, provides big-picture information on who uses it and
-why (or when), and lists prerequisites or other considerations.
+
+This command configures and runs the Splinter daemon, `splinterd`. This daemon
+provides core Splinter functionality such as circuit management, a REST API,
+and an admin service. It can also include application-specific services.
+
+The `splinterd` command requires a new unique ID for the node. Other settings
```

Comment:
How "unique" must the node ID be?  Across all possible node IDs on the network (aka The Internet)?  Just for the local Splinter node?  Across all current and future circuits?  

Also, what happens if the node ID is NOT unique?

Reply:
Unique among all nodes on a network. If using trust auth, two nodes could become confused for each other. (Hence why trust is insecure)

Reply:
What is a network? I thought there was no concept of a "splinter network"?

Reply:
I am using network to mean every possible splinterd you can connect to from your splinterd. It is not a "real concept". Im sorry for the confusion.

Reply:
Addressed - removed "unique" from this phrase, and expanded the `--node-id` description to clarify what "unique" means.